### ISSUE TITLE
When i add a new layer to VSSM, the training process has the "Parameter indices" error?

### ISSUE THREAD

**Issue by zhangye-zoe** [MzeroMiko/VMamba#229]
I want to add a new network layer to class VSSM(nn.Module), like following:

<img width="793" alt="截屏2024-06-11 下午9 58 47" src="https://github.com/MzeroMiko/VMamba/assets/48485190/f0c6a10e-52f8-437b-a9a5-59672f007bfa">

But when training the network, which has the following error:
<img width="1462" alt="截屏2024-06-11 下午10 00 34" src="https://github.com/MzeroMiko/VMamba/assets/48485190/b6bc4042-2194-40df-b293-599263be755e">

Could you give me some advice to solve the problem, thank you!

**Comment by MzeroMiko** [comment]
Please ensure that all the parameters are involved when calculating loss, otherwise, some parameters may not recieve gradients.
